joomla/joomla-cms
# [4.0][RFC] Custom Element: Joomla! Toolbar button

This is redo of #19635 

This patch add `<joomla-toolbar-button/>` custom element, to standard toolbar button.
It allow us to move most of inline onclick="" to separate file.

Currently it only for `StandardButton`.

### Testing Instructions
Click the toolbar buttons here and there, and make sure all works as expected, as before the patch.



ping @dgrammatiko @wilsonge
